.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,6 +1,9 @@
 ---
 name: Bug report
 about: Create a report to help us improve OpenRefine
+title: ''
+labels: bug, triage needed
```

Comment:
Is there any difference between `triage needed` and `to be reviewed` or should we merge these two tags?

Reply:
Happy to use the historical `to be reviewed`. I just wasn't sure of the current semantics.

Reply:
Ok, that's pretty much how we have been using it so far, so let's do that.